ext/io_gdml.jl

```diff
@@ -280,8 +317,71 @@ function parse_geometry(e::Ellipsoid{T,<:Any, TR, Nothing, Nothing}, x_solids::X
     nothing
 end
 
-function parse_geometry(e::World{T, 3, Cylindrical}, x_solids::XMLElement, x_define::XMLElement, id::Integer, pf::AbstractString, v::Bool)::Nothing where {T}
+@inline parse_θ(::Type{T}, θ::Nothing) where {T} = (T(0), T(360), "deg")
+@inline parse_θ(::Type{T}, θ::Tuple{T, T}) where {T} = throw(AssertionError("Torus: θ values are currently not supported"))
```

Comment:
Where is this ever used?
(If not at all: maybe comment this out, so we can uncomment this later)